Directory.Build.props

```diff
@@ -15,7 +15,7 @@
     <AnalysisMode>Default</AnalysisMode>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(VisualStudioVersion)' == '17.0' ">
+  <PropertyGroup Condition=" '$(BuildingInsideVisualStudio)' == 'true' AND '$(VisualStudioVersion)' == '17.0' ">
```

Comment:
Could someone remind me why we did this? Is this because of all of the failures to load the analyzer dll?

Reply:
Yes. https://github.com/dotnet/aspnetcore/pull/34664 And I didn't verify the analyzers still ran on builds outside of VS 😞